src/map/skill.cpp

```diff
@@ -14464,7 +14464,6 @@ int32 skill_castend_pos2(struct block_list* src, int32 x, int32 y, uint16 skill_
 		case MO_BODYRELOCATION:
 		case CR_CULTIVATION:
 		case HW_GANBANTEIN:
-		case LG_EARTHDRIVE:
```

Comment:
If you remove it here, will it not send clif_skill_poseffect?

Reply:
> If you remove it here, will it not send clif_skill_poseffect?

It will not reach this code anymore as it's no longer a unit skill.